test/lib/ansible_test/_data/sanity/pylint/plugins/deprecated.py

```diff
@@ -19,7 +19,8 @@
               "or AnsibleModule.deprecate",
               "ansible-deprecated-version",
               "Used when a call to Display.deprecated specifies a version "
-              "less than or equal to the current version of Ansible",
+              "less than or equal to the current version of Ansible resp. "
+              "this collection",
```

Comment:
Instead of making this message apply to both Ansible and collections, why not have two error codes/messages, one for each scenario?

Reply:
Implemented in e86cae662ded639828c8828aae177b47a1244a74.